modules/platform/login/nuxeo-platform-login-keycloak/src/main/java/org/nuxeo/ecm/platform/ui/web/keycloak/KeycloakAuthenticationPlugin.java

```diff
@@ -177,9 +177,10 @@ private KeycloakUserInfo getKeycloakUserInfo(AccessToken token) {
      */
     private Set<String> getRoles(AccessToken token, String keycloakNuxeoApp) {
         Set<String> allRoles = new HashSet<>();
-        Set<String> roles = token.getRealmAccess().getRoles();
-        if (roles != null) {
-            allRoles.addAll(roles);
+
+        AccessToken.Access realmAccess = token.getRealmAccess();
+        if (realmAccess != null && realmAccess.getRoles() != null) {
+            allRoles.addAll(realmAccess.getRoles());
         }
```

Comment:
The 10.10 commit has an added empty line after this `}`, please make sure your commits are not different without a reason.